posts/frangeris/03.secure-values-by-private-keys-in-aws.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Ensuring secure values by private keys in AWS (KMS, SSM, Secrets Manager)
+published: false
+description: Secure way of store and access sensible values on AWS
+tags: "aws, private keys security, serverless security, cloud security"
+cover_image: ./assets/03.cover.png
+---
+
+## Ensuring secure values by private keys in AWS (KMS, SSM, Secrets Manager)
+
+Private keys, credentials access and sensitive information such as passwords or any kind of confidential data are essentials in applications logic when interacting with other services or even databases, storing those values can be risky if proper security measures are not in place.
```

Comment:
```suggestion
Private keys, access credentials and sensitive information such as passwords or any kind of confidential data are essential in applications' logic when interacting with other services or even databases. Storing those values can be risky if proper security measures are not in place.
```

Reply:
this should be `application's logic`, small typo here

Reply:
Thought you meant in general -> plural